include/language.php

```diff
@@ -628,16 +630,17 @@ public function get_locale( $filter = 'raw' ) {
 	 *
 	 * @since 3.4
 	 *
-	 * @param bool $raw Whether or not properties should be raw. Default to `false`.
+	 * @param string $context Whether or not properties should be filtered. Accepts `raw` or `display`.
+	 *                        Default to `display` which filters some properties.
 	 *
 	 * @return array Array of language object properties.
 	 *
 	 * @phpstan-return LanguageData
 	 */
-	public function get_object_vars( $raw = false ) {
+	public function get_object_vars( $context = 'display' ) {
 		$language = get_object_vars( $this );
 
-		if ( ! $raw ) {
+		if ( 'display' === $context ) {
```

Comment:
I would invert the logic. Indeed, in most cases, we want to use the methods and not send the internal properties. So unknown values should not return the internal properties.
```suggestion
		if ( 'raw' !== $context ) {
```

Reply:
Done in 2b4c5d4, as well as the change for `db`.